device-posture/device-posture.https.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+//<script src="../resources/orientation-event-helpers.js"></script>
+<script>
+'use strict';
+
+async function cleanup() {
+  await test_driver.set_device_posture('continuous');
```

Comment:
This will always reset the posture to the same value even if it was something else by default. You need to cache the original value and restore it instead.